source/braille.py

```diff
@@ -2384,7 +2385,7 @@ def message(self, text):
 		if (
 			not self.enabled
 			or config.conf["braille"]["showMessages"] == ShowMessages.DISABLED
-			or text is None
+			or text is None or config.conf["braille"]["mode"] == BrailleMode.SPEECH_OUTPUT.value

```

Comment:
New line for readability

```suggestion
			or text is None
			or config.conf["braille"]["mode"] == BrailleMode.SPEECH_OUTPUT.value
```